pco/src/delta.rs

```diff
@@ -125,37 +125,34 @@ fn lookback_hash_lookup(
   }
 }
 
+#[inline(never)]
 fn lookback_compute_goodness<L: Latent>(
   l: L,
   i: usize,
   latents: &[L],
   proposed_lookbacks: &[usize; PROPOSED_LOOKBACKS],
   lookback_counts: &mut [u32],
-  goodnesses: &mut [Bitlen; PROPOSED_LOOKBACKS],
-) {
+) -> usize {
+  let mut best_goodness = 0;
+  let mut best_lookback: usize = 0;
   for lookback_idx in 0..PROPOSED_LOOKBACKS {
```

Comment:
I think we just index into the proposed lookbacks with this? I believe if you run cargo clippy it will give you a lint

Reply:
Changed